### PR TITLE
11.0.0.CR1

### PR DESCRIPTION
**Pull request by tristantarrant**
https://github.com/infinispan/infinispan.github.io/issues/11.0.0.CR1

### CODE REVIEW

File: _config/ispn.yml [hunk 1]
```diff
@@ -32,16 +32,16 @@ stable:
...
-   maven_latest: 11.0.0.Dev05
-   release_date: 2020/05/01
-   fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12345476
+   version: 11.0.0.CR1
+   maven_latest: 11.0.0.CR1
+   release_date: 2020/05/24
+   fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12344573
    codename: N/A
-   whats_new: Indexing improvements, non-blocking everywhere, etc...
+   whats_new: XSite IRAC, non-blocking everywhere, server security usability, etc...
```
Comment (pruivo) [inline _config/ispn.yml:40]: "XSite IRAC" => XSite/Cross Site improvements
I thought we are going to use the word "IRAC" only internally and in the code.

File: blog/2020-05-25-infinispan-11.0.0.CR1.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+* The server is now secure by default
+* New encrypted properties realm
+* Simplified authentication and authorization configuration to be as automatic as possible
+
+== Other Server changes
+* Datasources
+* Runtime logging configuration
+* Server report
+
+== CLI
```
Comment (pruivo) [inline blog/2020-05-25-infinispan-11.0.0.CR1.adoc:61]: don't you want to mention the "report" command?

File: blog/2020-05-25-infinispan-11.0.0.CR1.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+tags: [ "release" ]
+---
+
+= Infinispan 11.0.0.CR1
+
+Dear Infinispan community,
+
+we've just released 11.0.0.CR1 which comes loaded with quite a few features and improvements.
+
+== Cross-Site Replication Improvements
```
Comment (pruivo) [inline blog/2020-05-25-infinispan-11.0.0.CR1.adoc:15]: this seems a lot of text comparing with the other sections. 
What about only add the "sub-titles"
```
* Support for multiple site masters
* Conflict detection and resolution for Asynchronous Cross-Site Replication
```

Comment (tristantarrant) [reply blog/2020-05-25-infinispan-11.0.0.CR1.adoc:15]: It is the highlighted feature. I'll leave it like that.

**Pull request merged**